core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.html

```diff
@@ -16,9 +16,14 @@
   </div>
   <div class="oppia-opportunities-progress oppia-animate-fadein" *ngIf="progressBarRequired">
     <div class="oppia-opportunities-progress-bar">
-      <div [hidden]="opportunityDataIsLoading" class="progress">
+      <div [hidden]="opportunityDataIsLoading" *ngIf="!isTranslationProgressBar" class="progress">
         <div class="progress-bar bg-success" role="progressbar" [ngStyle]="progressBarStyle"></div>
       </div>
+      <div [hidden]="opportunityDataIsLoading"  *ngIf="isTranslationProgressBar" class="progress">
+        <div class="progress-bar bg-success" role="progressbar" [ngStyle]="translatedProgressStyle" ngbTooltip="accepted({{opportunity.translationsCount}}/{{opportunity.totalCount}})" container="body" placement="bottom"></div>
+        <div class="progress-bar bg-light-green" role="progressbar" [ngStyle]="inReviewProgressStyle" ngbTooltip="waiting review({{opportunity.inReviewCount}}/{{opportunity.totalCount}})" container="body" placement="bottom"></div>
+        <div class="progress-bar bg-light" role="progressbar" [ngStyle]="untranslatedProgressStyle" ngbTooltip="untranslated({{opportunity.totalCount - (opportunity.translationsCount + opportunity.inReviewCount)}}/{{opportunity.totalCount}})" container="body" placement="bottom"></div>
+      </div>
       <lazy-loading [hidden]="!opportunityDataIsLoading"></lazy-loading>
     </div>
     <div class="oppia-opportunities-progress-label">
```

Comment:
Can we also add a tooltip to the percentage with the following format:

i.e. "Percentage of accepted translated cards. X/Y cards remaining."

Reply:
Done

Reply:
Can you post updated screenshots of the each tooltip?